### ISSUE TITLE
URLs with "=" do not open in Windows, opens explorer folder instead

### ISSUE THREAD

**Issue by prashantv** [rauschma/openurl#2]
When trying to use openurl to open a link with equals, eg, "http://www.google.com/test=a", openurl ends up opening an explorer window instead of the browser.

I found a alternate way to open URLs with special characters in Windows that you may want to use instead:

```
  var exec = require("child_process").exec
  execStr = "explorer.exe ^\"" + url + "^\"";
  exec(execStr);
```

**Comment by sp00x** [comment]
This was never fixed? Ran into it today when trying to use this module..

**Comment by jmc-88** [comment]
No, I think this was never fixed. The pull request (https://github.com/rauschma/openurl/pull/3) I filed almost three years ago has never received any reply.

**Comment by kaelzhang** [comment]
@jmc-88 `cmd -c start` is the very better way

**Comment by BrianGenisio** [comment]
Can you merge and publish PR (https://github.com/rauschma/openurl/pull/3)?  If you'd rather not maintain this anymore, can you transfer this to someone who will? (I'd be happy to)

**Comment by Interagierende-Systeme** [comment]
Hi everyone,
if anyone is still facing this problem, we published a fork to solve this problem as well: 
https://github.com/Interagierende-Systeme/openurl2
